src/rai_extensions/rai_perception/rai_perception/components/service_utils.py

```
@@ -0,0 +1,136 @@
+
+    Args:
+        connector: ROS2 connector with node
+        service_name: Service name to check
+        timeout_sec: Timeout for checking service availability (default: 0.1)
+
+    Returns:
+        True if service is available, False otherwise
+    """
+    try:
+        wait_for_ros2_services(connector, [service_name], timeout=timeout_sec)
+        return True
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)